test/core/end2end/fuzzers/api_fuzzer.cc

```diff
@@ -795,13 +846,28 @@ DEFINE_PROTO_FUZZER(const api_fuzzer::Msg& msg) {
             g_channel = grpc_insecure_channel_create(
                 action.create_channel().target().c_str(), args, nullptr);
           }
+          g_channel_actions.clear();
+          for (int i = 0; i < action.create_channel().channel_actions_size();
+               i++) {
+            g_channel_actions.push_back({
+                std::min(action.create_channel().channel_actions(i).wait_ms(),
```

Comment:
maybe add a:
`const ChannelAction& channel_action = action.create_channel().channel_actions(i);`
just above this, so that we can write `channel_action.wait_ms()`, `channel_action.add_n_bytes_writable()`

Reply:
I changed this in the latest commit.